Adds dashboard template and script. (#909)
google/kf: dashboards/README.md

# Kf Dashboards

### Script

> The [create-dashboard.py](create-dashboard.py) script is a python script that

Reviewer: ```suggestion
The [create-dashboard.py](create-dashboard.py) script is a Python script that
```

will use the [dashboard-template.json](dashboard-template.json) file to create

> a template. It uses the currently targeted GCP project (e.g., `gcloud config

Reviewer: ```suggestion
a template. It uses the currently targeted Google Cloud project (e.g., `gcloud config
```

get-value project`).

> It takes the following 3 arguments:

Reviewer: ```suggestion
It takes the following three arguments:
```

> 1. Dashboard name - The name of the dashboard.

Reviewer: ```suggestion
1. Dashboard name - The name of the dashboard that will be created.
```

Does this have some limits e.g. being a valid DNS identifier?

> 1. Cluster name - The name of the Kubernetes cluster.

Reviewer: ```suggestion
1. Cluster name - The name of the Kubernetes cluster the dashboard will target.
```

> 1. Space - The name of the Kf space.

Reviewer: ```suggestion
1. Space - The name of the Kf Space the dashboard will target.
```
